public/app/features/explore/state/actions.ts

```diff
@@ -182,11 +183,21 @@ export const updateTimeRange = (options: {
   exploreId: ExploreId;
   rawRange?: RawTimeRange;
   absoluteRange?: AbsoluteTimeRange;
+  syncedTimes?: boolean;
 }): ThunkResult<void> => {
-  return dispatch => {
-    dispatch(updateTime({ ...options }));
-    dispatch(runQueries(options.exploreId));
-  };
+  if (options.syncedTimes) {
```

Comment:
As we already have access to `state.syncedTimes` from inside the redux it seems like we do not need it as an argument to the action creator. Would prevent anyone from forgetting that argument somewhere and creating a bug that way.